### PR TITLE
TravisCI: tests

### PR DESCRIPTION
**Pull request by YevhenShBolt**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,17 @@
-.DS_Store
-.idea
\ No newline at end of file
+*
```
Comment (daisy1754) [inline .gitignore]: why you need these? if you use modman you don't have to put plugin within magento dir

File: app/code/community/Bolt/Boltpay/Helper/Data.php [hunk 2]
```diff
@@ -332,7 +332,13 @@ public function getItemImageUrl($item)
...
         $_product = $item->getProduct();
 
-        $image = $imageHelper->init($_product, 'thumbnail', $_product->getThumbnail());
+        $image = '';
+        try {
+            if ($_product->getThumbnail()) {
+                /** @var Mage_Catalog_Helper_Image $image */
+                $image = $imageHelper->init($_product, 'thumbnail', $_product->getThumbnail());
+            }
+        } catch (Exception $e) {  }
```
Comment (daisy1754) [inline app/code/community/Bolt/Boltpay/Helper/Data.php]: can you add comment here explaining what exception we ignore here?

Comment (YevhenShBolt) [reply app/code/community/Bolt/Boltpay/Helper/Data.php]: This is part of code from here - https://github.com/BoltApp/bolt-magento1/pull/238/files
by @johnmccombs1 .

File: .gitignore (hunk 1 above)
Comment (YevhenShBolt) [reply .gitignore]: It was there. It ignores all files except Bolt module files.

**Comment by daisy1754** [comment]
🎉

**Pull request merged**